src/test/regress/citus_tests/common.py

```diff
@@ -89,20 +104,28 @@ def start_databases(pg_path, rel_data_path, node_name_to_ports, logfile_prefix,
 
         subprocess.run(command, check=True)
 
+    parallel_run(start, node_name_to_ports.keys())
+
+    # We don't want parallel shutdown, here because that will fail when it's
```

Comment:
I think the comma is in the wrong place

Reply:
apparently no comma is needed at all in english: https://www.grammarly.com/blog/comma-before-because/